lib/rubocop/server/core.rb

```diff
@@ -71,15 +85,18 @@ def server_mode?
         true
       end
 
-      def start_server(host, port)
-        @server = TCPServer.open(host, port)
-
+      def output_stream
         # JSON format does not expected output message when IDE integration with server mode.
         # See: https://github.com/rubocop/rubocop/issues/11164
-        return if use_json_format?
+        return nil if use_json_format?
```

Comment:
Is this change intentional?

Reply:
I think it was just force of habit, I forgot the two are equivalent. I'll revert it since the older line is more concise.